subnet/etcdv2/local_manager.go

```diff
@@ -233,32 +239,63 @@ func (m *LocalManager) tryAcquireLease(ctx context.Context, config *Config, extI
 	}
 }
 
-func (m *LocalManager) allocateSubnet(config *Config, leases []Lease) (ip.IP4Net, error) {
+func (m *LocalManager) allocateSubnet(config *Config, leases []Lease) (ip.IP4Net, *ip.IP6Net, error) {
 	log.Infof("Picking subnet in range %s ... %s", config.SubnetMin, config.SubnetMax)
+	if config.EnableIPv6 {
+		log.Infof("Picking ipv6 subnet in range %s ... %s", config.IPv6SubnetMin, config.IPv6SubnetMax)
+	}
+
+	type IPs struct {
+		ip  ip.IP4
+		ip6 *ip.IP6
```

Comment:
Why you treat IP6 as a pointer and not IP4?

Reply:
the IP field in struct IP6Net is also a `*ip.IP6` ; so it just aligns with that